idl/idl-parser/src/org/jetbrains/dukat/idlParser/util.kt

```diff
@@ -0,0 +1,14 @@
+package org.jetbrains.dukat.idlParser
+
+import org.antlr.v4.runtime.ParserRuleContext
+import org.antlr.v4.runtime.tree.ParseTree
+import org.antlr.v4.runtime.tree.TerminalNode
+import org.antlr.webidl.WebIDLLexer
+
+internal fun List<ParseTree>?.filterIdentifiers(): List<ParseTree> = this?.filter {
+    it is TerminalNode && it.symbol.type == WebIDLLexer.IDENTIFIER_WEBIDL
+} ?: emptyList()
+
+internal fun getName(ctx: ParserRuleContext) = ctx.children.filterIdentifiers().first().text
```

Comment:
Here, on the other side, ParserRuleContext looks like a perfect fit for extending. 
Also, can you please add explicit return type = it's not immediately  obvious whether one should expect here something nullable or not.